client/src/utils/unitHelpers.ts

```diff
@@ -70,8 +71,9 @@ export function unparseRam(value: number) {
     };
 }
 
-export function parseCpu(value?: string) {
+export function parseCpu(value?: string | {string: string}) {
```

Comment:
I don't see where this change is being used. Is this still needed?

Reply:
Same as above